### DISCUSSION TITLE
Google Chrome crashes after using undetected_chromedriver.

### DISCUSSION THREAD

**Discussion by angelvt71** [ultrafunkamsterdam/undetected-chromedriver#1968] [Q&A]
Google Chrome crashes after using undetected_chromedriver as uc.Chrome(). I'm using 3.5.5 version for undetected_chromedriver and 127.0.6533.73 (Build oficial) (x86_64) version for Chrome. It crashes after entering into any page and doing whatever, getting into a blank page. I've tried to uninstall and install again chrome and also undetected_chromedrive. I'm using Mac OS Sonoma 14.5. Idk if it's related to the version of Chrome uc is trying to access and the one I have in my pc.
IT'S IMPORTANT TO SOLVE THIS AS SOON AS POSSIBLE. Thank you!

Also, I receive this right after using uc.Chrome(), and I manually need to press any of them:
<img width="1295" alt="Captura de pantalla 2024-07-29 a las 20 40 34" src="https://github.com/user-attachments/assets/f42cdcf8-7abb-45ea-bdfb-cb3f3ef14333">

UPDATE: In my case I solved it by adding this: 

options = Options()
options.add_argument("--disable-dev-shm-usage")
options.add_argument("--disable-gpu") 

browser = uc.Chrome(driver_executable_path=r"/Users/XXXXXXX/Downloads/chromedriver-mac-x64/chromedriver", options=options)

**Comment by trento322** [comment]
I was having the same issue, but I think I figured out a solution. Whatever the most recent Google Chrome update there was a bug. The problem was fixed by downgrading Chrome to an older version. Go to https://www.google.com/chrome/ and at the bottom click Other Platforms and choose Mac 10.13-10.14 and install it. It worked for me.
